src/connection-manager/index.ts

```diff
@@ -31,7 +32,8 @@ const defaultOptions: Partial<ConnectionManagerInit> = {
   maxEventLoopDelay: Infinity,
   pollInterval: 2000,
   autoDialInterval: 10000,
-  movingAverageInterval: 60000
+  movingAverageInterval: 60000,
+  inboundConnectionThreshold: 5
```

Comment:
This will deny any host not in the allow list that tries to open more than 5x connections a second.  I plucked the number out of thin air, is it reasonable?